styles/main.css

```diff
@@ -399,28 +400,28 @@ details[open] > summary::before {
 
 .contact form {
   justify-content: center;
-  color: #ebebff;
+  color: #595959;
   width: 100%;
 }
 
 .contact h2 {
-  color: #fff;
+  color: #595959;
   text-align: center;
   font-size: 36px;
   line-height: 40px;
   margin-top: 0;
   margin-bottom: 10px;
 }
 
-.contact h4 {
+.contact h3 {
   height: 24px;
   font-size: 18px;
   font-weight: 500;
   margin-bottom: 0;
 }
 
 .contact p {
-  color: #ebebff;
+  color: #595959;
   text-align: center;
```

Comment:
- [ ] Kindly you can reduce the contrast and the color of the text in the paragraph. This will increase the readability of the text as the background has a different color.
![Screenshot from 2021-12-15 00-52-31](https://user-images.githubusercontent.com/65068771/146094579-02feff1a-c0e9-407f-9714-d6a0842f9f7b.png)

Reply:
Done, thanks :-)